packages/story-editor/src/components/footer/layers/layers.js

```diff
@@ -42,7 +42,9 @@ const StyledNavigationWrapper = styled(NavigationWrapper)`
 `;
 
 function Layers() {
-  const layersLength = useLayers().length;
+  const nunElements = useStory(
+    ({ state }) => state.currentPage?.elements?.length
+  );
```

Comment:
@miina, well-spotted - I didn't realize this used the same hook before. Well, now that layers mean all the rows in the layer panel (both elements and groups), I feel it makes sense for this to just show the number of elements rather than the full number of layers, or?

Reply:
Makes sense to me!